packages/fxa-email-service/scripts/test_with_authdb.sh

```diff
@@ -1,18 +1,16 @@
 #!/bin/sh
 
-DB_REPO=fxa-auth-db-mysql
+set -e
 
-if [ ! -e "$DB_REPO" ]; then
-  git clone "https://github.com/mozilla/$DB_REPO.git"
-fi
+DB_REPO=fxa-auth-db-mysql
 
 RUNNING_DB_SERVERS=`ps -ef | grep "[n]ode bin/server" | wc -l`
 if [ "$RUNNING_DB_SERVERS" -eq "0" ]; then
-  cd "$DB_REPO"
-  npm i
+  cd "../$DB_REPO"
```

Comment:
Would this have a similar issue as https://github.com/mozilla/fxa/pull/778#discussion_r273531109 and break fxa-dev?

Reply:
Ah-ha, hoist by my own petard I guess. In theory it would, although nobody should ever need to run this script there because the auth db is already running. The tests run standalone without needing this script if an auth db instance is running.

But it's a fair point, so I'll address this along with the other feedback.